### PR TITLE
added header content

### PR DESCRIPTION
**Pull request by ilyabaranowski**

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -6,69 +6,101 @@
...
-            <div class="offer__items">
-                <div class="offer__item">
-                    <img src="../Triphouse-project/images/earphones.svg" alt="headphones" />
-                    <span>Support 24/7</span>
+    <div class="wrapper">
+        <header>
+            <div class="header">
+                <div class="header__top">
+                    <img src="../Triphouse-project/images/header_triphouse_logo.svg" alt="triphouse-logo">
+                    <div class="header__navigation">
```
Comment (ghost) [inline Triphouse-project/index.html:18]: 2021-04-16_00h06_59
You should wrap that divs.

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -6,69 +6,101 @@
...
-                    <img src="../Triphouse-project/images/conversation.svg" alt="conversatiom" />
-                    <span>Communicate directly</span>
+                <div class="header__title">
+                    <h2>Discover stays <br /> live, work or just relax</h2>
                 </div>
-                <div class="offer__item">
-                    <img src="../Triphouse-project/images/pc.svg" alt="pc image">
-                    <span>Book online</span>
+                <div class="header__appstore-links">
+                    <span><a href="#"><img src="../Triphouse-project/images/header_Gplay.svg"
```
Comment (ghost) [inline Triphouse-project/index.html]: Span not needed.

File: Triphouse-project/index.html [hunk 3]
```diff
@@ -6,69 +6,101 @@
...
+                                alt="playmarket"></a></span>
+                    <span><a href="#"><img src="../Triphouse-project/images/header_Apple.svg" alt="appstore"></a></span>
```
Comment (ghost) [inline Triphouse-project/index.html]: Span not needed.

File: Triphouse-project/index.html [hunk 4]
```diff
@@ -6,69 +6,101 @@
...
-            <h2>What do we offer</h2>
-            <div class="offer__items">
-                <div class="offer__item">
-                    <img src="../Triphouse-project/images/earphones.svg" alt="headphones" />
-                    <span>Support 24/7</span>
+    <div class="wrapper">
+        <header>
+            <div class="header">
+                <div class="header__top">
+                    <img src="../Triphouse-project/images/header_triphouse_logo.svg" alt="triphouse-logo">
```
Comment (ghost) [inline Triphouse-project/index.html]: images shout be in Triphouse-project folder

File: Triphouse-project/index.html (hunk 1 above)
Comment (ilyabaranowski) [reply Triphouse-project/index.html:18]: done

File: Triphouse-project/index.html (hunk 2 above)
Comment (ilyabaranowski) [reply Triphouse-project/index.html]: deleted

File: Triphouse-project/index.html (hunk 3 above)
Comment (ilyabaranowski) [reply Triphouse-project/index.html]: deleted

File: Triphouse-project/index.html (hunk 4 above)
Comment (ilyabaranowski) [reply Triphouse-project/index.html]: replaced to Triphouse-project folder

**Pull request merged**